apps/portals/standards/src/config/synapseConfigs/data.ts

```diff
@@ -12,8 +12,8 @@ export const dataColumnLinks: LabelLinkConfig = [
   {
     isMarkdown: false,
     baseURL: 'Explore/Standard/DetailsPage',
-    URLColumnName: 'id',
-    matchColumnName: 'id',
+    URLColumnName: 'acronym',
```

Comment:
Are we going to use acronym (instead of id) for the fk relationship in other tables on the details page?